scripts/install

```diff
@@ -162,27 +161,25 @@ done
 #=================================================
 # SECURE FILES AND DIRECTORIES
 #=================================================
+ynh_script_progression --message="Securing files and directories..."
 
 # Set permissions to app files
-chown -R root: $final_path
-chown pepettes:root $final_path
-chown pepettes:root $final_path/settings.py
-chmod o=--- $final_path/settings.py
```

Comment:
How we avoid to expose this files to others users (like potential hacked apps)

Reply:
Did a `chmod -R 750` or `chmod -R 770` would be enough ?
- `chmod -R 750` : https://chmodcommand.com/chmod-750/
- `chmod -R 770` : https://chmodcommand.com/chmod-770/
In that case only the $app has access and also root group